assets/src/edit-story/elements/image/karma/resourceLoading.karma.js

```diff
@@ -33,9 +33,10 @@ describe('Image resource loading integration', () => {
     fixture.restore();
   });
 
-  // Disable reason: flaky and blocks merging. See https://github.com/google/web-stories-wp/issues/4388
```

Comment:
If this change fixes #4388, please update the PR description to say "Fixes #4388"

Reply:
Done.